source/gui/settingsDialogs.py

```diff
@@ -4145,6 +4157,17 @@ def makeSettings(self, settingsSizer):
 		self.bindHelpEvent("BrailleSettingsWordWrap", self.wordWrapCheckBox)
 		self.wordWrapCheckBox.Value = config.conf["braille"]["wordWrap"]
 
+		self.unicodeNormalizationCombo: nvdaControls.FeatureFlagCombo = sHelper.addLabeledControl(
+			labelText=_(
+				# Translators: This is a label for a combo-box in the Braille settings panel.
+				"Unicode normali&zation"
+			),
+			wxCtrlClass=nvdaControls.FeatureFlagCombo,
+			keyPath=["braille", "unicodeNormalization"],
+			conf=config.conf,
+		)
+		self.bindHelpEvent("BrailleSettingsUnicodeNormalization", self.unicodeNormalizationCombo)

```

Comment:
Anchor names should not contain the panel they are defined in, in case they are moved later.

```suggestion
		self.bindHelpEvent("BrailleUnicodeNormalization", self.unicodeNormalizationCombo)
```